File: cpp/breakContinue.cpp
#include <iostream>
using namespace std;

int main()
{
	cout<<"break at 5"<<endl;			//at 5 loop stops
    for(int i=0; i<10; i++){
        cout<<i<<endl;
        if(i==5)
            break;
    }
    int i=0;
    
    cout<<endl<<"continue at 5"<<endl;	//at 5 loop stops only once then start from 6
    while(i<10){
        i++;
        if(i==5)
            continue;
        cout<<i<<endl;
    }
}

File: java/breakContinue.java
package git;
public class breakContinue {
    public static void main(String str[]){
        for(int i = 0; i <= 10; i++){
            if(i==5){                         //break for only when i is 5
                continue;
            }
            if(i==8){
                break;
            }
            System.out.println(i);          //break loop permanently after i is 8
        }
    }
}
